Alpaca.Markets/Helpers/OpenClose.cs

```diff
@@ -20,7 +20,7 @@ public OpenClose()
     /// <param name="date">Session date</param>
     /// <param name="open">Open time in EST time zone.</param>
     /// <param name="close">Close time in EST time zone.</param>
-    internal OpenClose(
+    public OpenClose(
```

Comment:
I prefer to keep this constructor internal but I'm OK with adding public `init` accessors for `OpenEst` and `CloseEst` properties. It will not spoil public API with unnecessary details and will solve your problem with testability.